Explorer/Assets/Scripts/Global/ComponentsContainer.cs

```diff
@@ -69,6 +69,12 @@ public static ComponentsContainer Create()
                                                              onGet: SDKComponentBuilderExtensions.SetAsDirty,
                                                              onRelease: pbe => pbe.Reset())
                                                         .Build())
+               .Add(SDKComponentBuilder<PBVideoEvent>.Create(ComponentID.VIDEO_EVENT)
```

Comment:
As all the fields in the components are non-optional and value-types you don't need a custom registration (do you need a dirrty flag?) - you can use `AsProtobufResult`